css-brochure-form/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Brochure Form</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+  <body>
+    <h2>LEARN MORE</h2>
+    <form action="" method="get">
+      <div class="form">
+        <label for="email" id="email">COMPANY EMAIL</label>
+        <input type="text" placeholder="sally@company.co" required>
+      </div>
+
+      <div class="form">
+        <label for="name" id="name">NAME</label>
+        <input type="text" name="name" placeholder="Sally Smith"
+  required>
+      </div>
+
+      <div class="form">
+        <label for="location">WHERE ARE YOU HIRING?</label>
+        <input type="text" name="location" placeholder="City, Country" required>
+      </div>
+
+      <div class="form">
+        <label for="engineers" id="engineers">HOW MANY ENGINEERS ARE YOU HIRING IN THE NEXT SIX MONTHS?</label>
+        <select name="options">
```

Comment:
The `id` should be getting applied to the `select` rather than the `label`. You give a `label` a `for` attribute, that matches the `id` of an `select`.